### PR TITLE
feat: shorten the readme / package page docs

### PR DESCRIPTION
**Pull request by ryanswrt**
This PR moves the majority of what was in the readme into comprehensive walkthrough doc

### CODE REVIEW

**Review by seankhl — APPROVED** [review]
1. prepare_protein uses pdbfixer and pdb2pqr so we should note that
2. I realized that the "relative path fails, absolute path overwrites" thing is a bit confusing and nonstandard. Maybe we should have an optional "force" flag defaulting to false and make the filepath behave the same regardless of it being absolute or relative? Filename behavior putting in workdir can stay.
3. What's the SCALARS thing for?

File: README.md [hunk 1]
```diff
@@ -173,50 +120,10 @@ help(client.convert)
...
-
-We will now demonstrate how this works in action
-
-## 1.1) Input Preparation
-
-### 1.1.1) Prep the protein
+## 1.1) Prep the protein
 
 First we will run the protein preparation routine (using pdbfixer
 internally) to prepare the protein for molecular dynamics
```
Comment (seankhl) [inline README.md]: Add pdb2pqr mention here

File: rush/provider.py [hunk 2]
```diff
@@ -871,6 +890,7 @@ async def get_module_functions(
 
                 default_target = None
                 if module.targets:
+                    random.shuffle(module.targets)
```
Comment (seankhl) [inline rush/provider.py:898]: Why?

**Review by seankhl — COMMENTED** [review]
More comments

File: rush/provider.py (hunk 2 above)
Comment (ryanswrt) [reply rush/provider.py:898]: Answered with comment

**Pull request merged**